Makefile

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+# See LICENSE.txt for license information.
+
+## Docker Build Versions
+DOCKER_BUILD_IMAGE = golang:1.15.7
```

Comment:
Out of interest, is it necessary to stick on specific minor versions too? Is this a convention we have?

Reply:
It is usually a good practise to specify the version to control what is being used.

Reply:
tiny nit, but golang 1.15.8 is out now for us to use.